main.go

```diff
@@ -23,15 +25,67 @@ import (
 	"github.com/rakyll/statik/fs"
 )
 
+// Theme of longcat
+type Theme struct {
+	Head image.Image
+	Body image.Image
+	Tail image.Image
+}
+
 func loadImage(fs http.FileSystem, n string) (image.Image, error) {
 	f, err := fs.Open(n)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(fmt.Errorf("theme file does not open: %s", n))
 	}
 	defer f.Close()
 	return png.Decode(f)
 }
 
+func loadImageGlob(dir string, glob string) (image.Image, error) {
+	pattern := filepath.Join(dir, glob)
+	matches, err := filepath.Glob(pattern)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if len(matches) == 0 {
+		log.Fatal(fmt.Errorf("file does not exist: %s", pattern))
+	}
+
+	data, err := ioutil.ReadFile(matches[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	img, err := png.Decode(bytes.NewReader(data))
+	if err != nil {
+		log.Fatal(err)
+	}
+	return img, nil
+}
+
+func (t *Theme) loadTheme(themeName string) error {
+	fs, err := fs.New()
+	if err != nil {
+		return err
+	}
+
+	imgPath := func(s string) string {
+		return filepath.Join("/themes", themeName, s)
+	}
+
+	t.Head, _ = loadImage(fs, imgPath("data01.png"))
+	t.Body, _ = loadImage(fs, imgPath("data02.png"))
+	t.Tail, _ = loadImage(fs, imgPath("data03.png"))
+	return nil
+}
+
+func (t *Theme) loadThemeFromDir(dir string) error {
+	t.Head, _ = loadImageGlob(dir, "*1.png")
+	t.Body, _ = loadImageGlob(dir, "*2.png")
+	t.Tail, _ = loadImageGlob(dir, "*3.png")
+	return nil
+}
```

Comment:
When using statik, loadImage always be succeeded. But loadImageGlob possibly be fail. So should return err.

Reply:
I tried to fix it (tobynet/longcat@b1f04c9cfe2c724cc323ded29145a50eecac68d5)